time/README.md

```diff
@@ -0,0 +1,124 @@
+# Time - Getting Wall Time
+
+This example shows you how to use the Blocky Attestation Service (Blocky AS) to
+attest simple function call that gets the current wall clock time.
+
+Before starting this example, make sure you are familiar with the
+[Hello World - Attesting a Function Call](../hello_world_attest_fn_call/README.md),
+[Passing Input Parameters and Secrets](../params_and_secrets/README.md),
+and the
+[Error Handling - Attested Function Calls](../error_handling_attest_fn_call/README.md)
+examples.
+
+You'll learn how to:
+
+- Get the current wall time
+
+*Note: Wall time only works on Blocky AS versions 0.1.0-beta.6 and above.
+While this example may run on previous versions it will generate time based
+on a hardcoded start date.*
+
+## Setup
+
+- Install the Blocky AS CLI by following the
+  [setup instructions](https://blocky-docs.redocly.app/attestation-service/setup)
+  in the Blocky AS documentation.
+- Make sure you also have
+  [Docker](https://www.docker.com/) and [jq](https://jqlang.org/) installed on
+  your system.
+
+## Quick Start
+
+To run this example, call:
+
+```bash
+make run
+```
+
+You will see the following output extracted from a Blocky AS response:
+
+```
+Output:
+{
+  "Success": true,
+  "Error": "",
+  "Value": "2025-04-03T20:13:12.9481826Z"
+}
+```
+
+## Walkthrough
+
+### Step 1: Create a function for getting the current time
+
+By default, the `wazero` runtime does not provide sandboxed functions access to
+the system clock. Instead, they are given a hardcoded time that monotonically 
+increases by "1ns" for every subsequent call. In the Blocky AS environment,
+however, we have configured `wazero` to grant functions access to the system
+clock.
```

Comment:
Should I add that the system clock is either sourced from the enclave environment (i.e., Nitro) or from a remote time server?

Reply:
Probably worth adding that it comes from a remote time server.  We have another card https://blocky.atlassian.net/browse/BKY-6541 to update this example once we merge in the PTP updates into newman and delphi.

Reply:
updated to mention time is fetched from a remote server